server/src/internalClusterTest/java/org/opensearch/search/SearchTimeoutIT.java

```diff
@@ -82,7 +82,7 @@ protected Settings nodeSettings(int nodeOrdinal) {
     }
 
     public void testSimpleTimeout() throws Exception {
-        final int numDocs = 1000;
+        final int numDocs = 10;
         for (int i = 0; i < numDocs; i++) {
```

Comment:
This is super nit, but could we bring in with line with 1.x / 1.3 line (since we figured out the previous change has introduced flakyness), thank you:

```suggestion
        for (int i = 0; i < 32; i++) {
```